profile_bluesky/startup/instrument/devices/huber.py

```diff
@@ -70,6 +69,16 @@ class FourCircleDiffractometer(E4CV):
     energy_update_calc_flag = Component(Signal, value=1)
     energy_offset = Component(Signal, value=0)
 
+    # TODO: This is needed to prevent busy plotting.
+    @property
+    def hints(self):
+        fields = []
+        for _, component in self._get_components_of_kind(Kind.hinted):
+            if (~Kind.normal & Kind.hinted) & component.kind:
```

Comment:
Did not know about the bitwise inverse operator `~`.  Hrmph!

Reply:
Copied from `ophyd.signal.Signal` 😅

https://github.com/bluesky/ophyd/blob/7937cd46e92e9ec2f7eec321dd5b1b1b71115e20/ophyd/signal.py#L457